lsp/anakin_language_server.lua

```diff
@@ -8,6 +8,7 @@
 ---
 --- * Initialization: https://github.com/muffinmad/anakin-language-server#initialization-option
 --- * Configuration: https://github.com/muffinmad/anakin-language-server#configuration-options
+---@type vim.lsp.Config
```

Comment:
Need a blank line above this. Else the `---@type...` is part of the generated doc. Plus that's more "correct" anyway since a `@brief` doc describes the module, not any particular code in the module.